Dockerfile

```diff
@@ -15,6 +13,9 @@ RUN apt-get update \
   && groupadd -f -g ${GID} -r dbt && useradd -g dbt -l -m -r -u ${UID} dbt \
   && python3 -m pip install -U wheel
 
+ARG DBT_VERSION
+ARG DBT_ADAPTER
```

Comment:
`ARG` instructions invalidate the build cache for subsequent `RUN` instructions. Moving this below the run instruction above (which doesn't use either of these args) allows that layer to be re-used when building images for different adapters/versions.